src/features/events/components/EventSelectionCheckBox.tsx

```diff
@@ -0,0 +1,84 @@
+import { Box, Checkbox } from '@mui/material';
+import { useSelector, useStore } from 'react-redux';
+
+import { eventsSelected } from '../store';
+import { RootState } from 'core/store';
+import { ZetkinEvent } from 'utils/types/zetkin';
+
+interface EventSelectionCheckBoxProps {
+  eventList: ZetkinEvent[];
+  multiSelect?: boolean;
+}
+const EventSelectionCheckBox = ({
+  eventList,
+  multiSelect,
+}: EventSelectionCheckBoxProps) => {
+  const store = useStore<RootState>();
+  const selectedEvents = useSelector(
+    (state: RootState) => state.events.selectedEvents
+  );
+
+  const alreadyExists = selectedEvents.some(
+    (selectedEvent) => selectedEvent.id == eventList[0].id
+  );
+
+  const handleChange = (checked: boolean) => {
+    if (multiSelect) {
```

Comment:
I don't understand the purpose of `multiSelect`. Shouldn't the length of `eventList` indicate whether this is a checkbox that selects multiple, or a single, event?